app/scripts/modules/core/src/pipeline/manualExecution/layout/ManualExecutionFieldLayout.tsx

```diff
@@ -18,10 +18,11 @@ export class ManualExecutionFieldLayout extends React.Component<IFieldLayoutProp
 
     return (
       <div className="sp-margin-m-bottom">
-        <div className={'form-group'}>
+        <div className="form-group">
           {showLabel && (
-            <label className={'col-md-4 sm-label-right'}>
-              {label} {help}
+            <label className="col-md-4 sm-label-right break-word">
+              {label}
+              {required && <span>*</span>} {help}
```

Comment:
the `required` treatment and `break-word` class were both on the hand-rolled parameter layouts, and it didn't seem problematic to make them a thing for all the fields on this modal.